common/test/acceptance/tests/discussion/test_discussion.py

```diff
@@ -211,6 +211,7 @@ def test_page_accessibility(self):
         self.page.a11y_audit.config.set_rules({
             "ignore": [
                 'section',  # TODO: AC-491
+                'aria-required-children',  # TNL-5169, AC-534
```

Comment:
@alisan617 Why did you add these ignores as part of completing TNL-5169? It's odd because usually the ticket number is for the work that needs to be done in order to delete the ignore, but in this case TNL-5169 was closed with this PR.